src/hooks/useKirbyApi.js

```diff
@@ -50,10 +50,12 @@ const apiUri = (path) => {
  * @param {boolean} [options.revalidate=false] Skip cache look-up and fetch page freshly
  * @returns {Promise<object|boolean>} The page's data or `false` if fetch request failed
  */
-const getPage = async (id, { revalidate = false } = {}) => {
+const getPage = async (id, { revalidate = false, token = null } = {}) => {
```

Comment:
I prefer `undefined` instead of `null`. Thus, `{ revalidate = false, token }` should be sufficient.